examples/ipynbtests.sh

```diff
@@ -41,37 +34,34 @@ cp `basename $mistis` toy_mistis_1k_OPS1.nc
 ls *nc
 cd toy_model_mstis/
 date
-ipynbtest.py "toy_mstis_1_setup.ipynb" || testfail=1
-date
-ipynbtest.py "toy_mstis_2_run.ipynb" || testfail=1
-date
-ipynbtest.py "toy_mstis_3_analysis.ipynb" || testfail=1
-date
-ipynbtest.py "toy_mstis_4_repex_analysis.ipynb" || testfail=1
-#date
-#ipynbtest.py "toy_mstis_5_srtis.ipynb" || testfail=1
+py.test --nbval-lax --current-env -v \
+    toy_mstis_1_setup.ipynb \
+    toy_mstis_2_run.ipynb \
+    toy_mstis_3_analysis.ipynb \
+    toy_mstis_4_repex_analysis.ipynb \
+    || testfail=1
+
 cd ../toy_model_mistis/
 date
-ipynbtest.py "toy_mistis_1_setup_run.ipynb" || testfail=1
-date
 # skip toy_mistis_2_flux: not needed
-ipynbtest.py "toy_mistis_3_analysis.ipynb" || testfail=1
-date
+py.test --nbval-lax --current-env -v \
+    toy_mistis_1_setup_run.ipynb \
+    toy_mistis_3_analysis.ipynb \
+    || testfail=1
+
 cd ../tests/
 cp ../toy_model_mstis/mstis.nc ./
-ipynbtest.py --strict --show-diff "test_openmm_integration.ipynb" || testfail=1
-date
-ipynbtest.py --strict "test_snapshot.ipynb" || testfail=1
-date
-ipynbtest.py --strict "test_netcdfplus.ipynb" || testfail=1
-date
-ipynbtest.py --strict "test_cv.ipynb" || testfail=1
-date
-ipynbtest.py --strict "test_pyemma.ipynb" || testfail=1
-date
+py.test --nbval --current-env \
```

Comment:
> Was the `-v` consciously dropped here?

Yes, this was intentional. The verbose output is not very useful with nbval, because each test is just named `Cell 0`, `Cell 1`, etc. (This is what I liked much more about ipynbtest and its parsing of the Markdown headings.) I added the `-v` for the slower notebooks, because I'm pretty sure that before too long I'll know exactly which cells are expected to take a while, and it'll be nice to get line-by-line updates for them. For fast-running notebooks, I didn't see a need for it.